server/index.js

```diff
@@ -2,15 +2,38 @@ require('dotenv/config');
 const express = require('express');
 const staticMiddleware = require('./static-middleware');
 const errorMiddleware = require('./error-middleware');
+const axios = require('axios');
+const crypto = require('crypto');
```

Comment:
The new way to use node provided modules is `require('node:crypto')` or `import crypto from `node:crypto`.

Reply:
Is there a difference with this btw? I tried using both and they seemed to both work.

Reply:
No difference. The 'node:' prefix just makes it clear it's a built-in node module.